Library/Formula/postgresql.rb

```diff
@@ -72,13 +74,18 @@ def install
       ENV.append %w{CFLAGS LDFLAGS}, "-arch #{Hardware::CPU.arch_32_bit}"
     end
 
+    system "pkill postgres || true"
     system "./configure", *args
     system "make", "install-world"
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version}"
+    elsif (var/"postgres-#{version}").exist?
+      ohai "You're all set up and good to go, Awesome!"
```

Comment:
Wasn't really sure what to do here. Wanted to just stick a ` true ` in but wasn't sure if that would drag ` true ` across the entire install.